src/test/unit/requests/publicationRequests.test.ts

```diff
@@ -61,53 +59,128 @@ describe('Get publication by court id', () => {
   dataManagementStub.withArgs('/publication/courtId/error').resolves(Promise.reject(errorMessage));
 
   it('should return data on successful get', async () => {
-    expect(await publicationRequests.getPublicationsByCourt(valid, true)).toBe(successResponse.data);
+    expect(await pubRequests.getPublicationsByCourt(valid, true)).toBe(successResponse.data);
   });
 
   it('should handle error response from returned service returning empty array', async () => {
-    expect(await publicationRequests.getPublicationsByCourt(invalid, true)).toStrictEqual([]);
+    expect(await pubRequests.getPublicationsByCourt(invalid, true)).toStrictEqual([]);
   });
 
   it('should handle error request from returned service returning empty array', async () => {
-    expect(await publicationRequests.getPublicationsByCourt('test', true)).toStrictEqual([]);
+    expect(await pubRequests.getPublicationsByCourt('test', true)).toStrictEqual([]);
   });
 
   it('should handle error request from returned service returning empty array', async () => {
-    expect(await publicationRequests.getPublicationsByCourt('error', true)).toStrictEqual([]);
+    expect(await pubRequests.getPublicationsByCourt('error', true)).toStrictEqual([]);
   });
 });
 
 describe('get Publication request', () => {
-  dataManagementStub.withArgs('/publication/search/0').resolves({data: pubs});
+  dataManagementStub.withArgs('/publication/courtId/0').resolves({data: pubs});
```

Comment:
youre gunna need to check this file becuase this is where you merged into mine with your duplicated tests, we already have courtId tests so these are mostly repeated directly above